### PR TITLE
Small fix

### PR DESCRIPTION
**Pull request by Nanush7**
"For competition..." -> "For competitions..."

### CODE REVIEW

**Comment by Nevseros** [comment]
Are you sure this is needed? There is also `competition` in the end of the regulation.

**Comment by Nanush7** [comment]
The first one is "for all the competitions with multiple physical locations", and the other one is "to consider the competition as ended" (each competition with that condition). 9u is about the end of the competition, and in 9u+ we are making an addition to all those competitions that have multiple physical locations.

I'm not sure if this is correct, though.

**Comment by Rouxles** [comment]
i believe competitions here is correct

File: wca-guidelines.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ To be more informative, each Guideline is classified using one of the following
...
 - 9t++) [ADDITION] If the change of results happens during the competition, this rule must also be applied. In that case, [9t1](regulations:regulation:9t1), [9t2](regulations:regulation:9t2) and [9t3](regulations:regulation:9t3) should also be followed, at the discretion of the WCA Delegate.
 - 9t3+) [EXAMPLE] Suppose competitor A gets a 10.00 average and competitor B gets a 10.05 average in the first round of an event, such that competitor A advances to the final round but not B. If a score-taking mistake is discovered, and competitor B actually got a 9.95 average (and should have proceeded instead of A), competitor A keeps their result for the final round.
-- 9u+) [ADDITION] For competition with multiple physical locations (e.g. "3x3x3 Fewest Moves simultaneous competitions"), all the requirements must be met in every location to consider the competition as ended.
+- 9u+) [ADDITION] For competitions with multiple physical locations (e.g. "3x3x3 Fewest Moves simultaneous competitions"), all the requirements must be met in every location to consider the competition as ended.
```
Comment (ohexter) [inline wca-guidelines.md:131]:
```suggestion
- 9u+) [ADDITION] For a competition with multiple physical locations (e.g. "3x3x3 Fewest Moves simultaneous competitions"), all the requirements must be met in every location to consider the competition as ended.
```

Alternative option. I'm not too familiar with style conventions but this is consistent with 1a, 1b, 2a

Comment (Nevseros) [reply wca-guidelines.md:131]: We've already requsted the Board approval on this. The current one is still fine, right?

Comment (ohexter) [reply wca-guidelines.md:131]: It's fine

**Review by Nevseros — APPROVED** [review]

**Pull request merged**